Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -349,18 +349,18 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
       }
 
     let cachedFeatureFlagValue = self.sortProperty.signal.skipNil()
-      .map { _ in featureGoRewardlessIsEnabled() }
-    let updatedFeatureFlagValue = self.configUpdatedProperty.signal.skipNil()
-      .map { _ in featureGoRewardlessIsEnabled() }
+      .map { _ in editorialLightsOnFeatureIsEnabled() }
+    let updatedFeatureFlagValue = self.optimizelyClientConfiguredProperty.signal
```

Comment:
So I think this logic needs to change a little bit now that we're using Optimizely for feature flags instead of the config. I think you can just use `optimizelyReady` and remove `latestFeatureFlag` stuff, which was in place before because it was reading the feature flag from the `Config`.

Reply:
I was thinking it might help with race conditions if Optimizely takes time to configure and the data file changes resulting in the feature being enabled/disabled?